frontend/src/libs/components/link/styles.module.scss

```diff
@@ -0,0 +1,18 @@
+.link {
+  display: flex;
+  align-items: center;
+  padding: 5px 10 px;
```

Comment:
I don't see styles for link state in figma, so let's keep it simple for now: remove padding and background-color changing on hover

Reply:
I don't have the link to the figma yet

Reply:
in the slack calmpal_team channel we have all links pinned to the top. Please let me know if you still aren't able to get it
<img width="608" alt="Screenshot 2023-08-21 at 00 21 54" src="https://github.com/BinaryStudioAcademy/bsa-2023-calmpal/assets/64521658/7586e9c9-6583-4e47-bfc8-d1c4f065ce0e">